src/DraxList.tsx

```diff
@@ -573,7 +592,7 @@ export const DraxList = <T extends unknown>(
 	// Monitor drags to react with item shifts and auto-scrolling.
 	const onMonitorDragOver = useCallback(
 		(eventData: DraxMonitorEventData) => {
-			const { dragged, receiver, monitorOffsetRatio } = eventData;
+			const { dragTranslation, dragged, receiver, monitorOffsetRatio } = eventData;
```

Comment:
Minor style tweak here: for objects with more than 3 items, they should be broken onto separate lines.